File: Arcade/Intro/15_addBorder.js
function addBorder(picture) {
    
    for (var i = 0; i < picture.length; i++) {
        picture[i] = "*" + picture[i] + "*";
    }
    
    picture.unshift("*".repeat(picture[0].length));
    picture.push("*".repeat(picture[0].length));
    
    return picture;
}

File: Arcade/Intro/C++/15_addBorder.cpp
std::vector<std::string> addBorder(std::vector<std::string> picture) {
string topBottom;
    topBottom.assign(picture[0].length()+2,'*');
    for(string &s : picture){
        s="*"+s+"*";
    }
    picture.insert(picture.begin(),topBottom);
    picture.push_back(topBottom);
    return picture;
}
